src/common/debug_control.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+// Prevent system-wide input lagging while paused in the debugger
+//#define DISABLE_LOWLEVEL_KBHOOK_WHEN_DEBUGGED
```

Comment:
Since we also have a mouse hook, maybe rename to `DISABLE_LOWLEVEL_HOOKS_WHEN_DEBUGGED`